app/controllers/api/v2/helm_infrastructures_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V2::HelmInfrastructuresController < Api::V2::BaseController
+  def delete
```

Comment:
We can write this function under /app_groups_controller.rbfile instead. https://github.com/BaritoLog/BaritoMarket/blob/master/app/controllers/api/v2/app_groups_controller.rb